index.html

```diff
@@ -279,6 +279,22 @@ <h2 class="skill-name">Skills</h2>
             </ul>
         </div>
     </section>
-    
+    <section id="contact-me">
+        <h2 id="contact-me-header">Contact me</h2>
+        <p id="contact-me-description">If you have an application you are interested in developing, a feature that you need built or a project that needs
+        coding. I’d love to help with it</p>
+        <form id="contact-me-form" action="https://formspree.io/f/mpzbgqyj" method="post">
+            <div class="form-field">
+                <input required name="fullname" maxlength="30" type="text" placeholder="Enter your full name">
+            </div>
+            <div class="form-field">
+                <input required name="email" type="email" placeholder="Enter your email address">
+            </div>
+            <div class="form-field">
+                <textarea required name="message" maxlength="500" cols="30" rows="10" placeholder="Write your message here"></textarea>
```

Comment:
Great job @samlorlah 🤝

- [ ] [Optional] You may please try to make your design as identical as possible. Please add that line at the bottom of your contact form.
- Your design
![your contact](https://user-images.githubusercontent.com/63917725/177372934-c865d60e-85df-4423-9b83-d9d528786201.PNG)

- Figma design
![figma contact](https://user-images.githubusercontent.com/63917725/177373008-afa830fa-3578-425c-a6d4-6466f1187753.PNG)

Reply:
@1GORDON .

Thanks for approving.

However your optional suggestion is noted but it is not needed. That is a symbol of home button for mobile view which comes naturally with phones like iPhones with Face ID.

Once again, I appreciate.